app/helpers/application_helper/button/zone_collect_logs.rb

```diff
@@ -1,15 +0,0 @@
-class ApplicationHelper::Button::ZoneCollectLogs < ApplicationHelper::Button::LogDepotEdit
-  include ApplicationHelper::Button::Mixins::ButtonPromptMixin
```

Comment:
Interestingly, from what I can tell, ButtonPromptMixin is only used by ZoneCollectLogs and CollectLogs, so this mixin can _also_ be deleted 🗑️